eduvmstorebackend/requirements.txt

```diff
@@ -1,7 +1,7 @@
-Django
-djangorestframework
-openstacksdk
-sqlalchemy
-ruff
-vulture
-bandit
\ No newline at end of file
+Django==4.2
+djangorestframework==3.15.2
+openstacksdk==4.0.0
+sqlalchemy==2.0.35
+ruff==0.6.8
+vulture==2.12
+bandit==1.7.10
```

Comment:
sind ruff etc. wirklich requirements? 
sollten hier nicht nur sachen sein, welceh zum ausführen benötigt werden?

Reply:
letzendlich wahrscheinlich schon, jetzt ist es ganz praktisch um die Entwicklungsumgebung schnell aufzusetzen und finde stört aktuell auch nicht so.

Reply:
Werden die wohl in irgendeiner Art für die lokale Entwicklung benötigt werden? Ansonsten könnten wir sie jetzt oder zu einem späteren Zeitpunkt entfernen.